app/pages/patientdata/patientdata.less

```diff
@@ -33,16 +33,21 @@
 
 .patient-data-subnav-text {
   display: inline-block;
-  padding: @patient-data-subnav-vertical-padding @patient-data-subnav-horizontal-padding;
+  padding: @patient-data-subnav-vertical-padding 0 @patient-data-subnav-horizontal-padding 0;
```

Comment:
Lets get rid of the th after the #

*Sara Krugman *
Lead designer

*Tidepool *An open source, not-for-profit effort to build an open data
platform and better applications that reduce the burden of Type 1 Diabetes
and accelerate the commercialization of closed-loop systems.

*Phone :* 617 549 1483
*Email : *sara@tidepool.org
*Web : *Tidepool.org <http://tidepool.org/>




On Tue, Jan 13, 2015 at 10:24 AM, Jana Beck <notifications@github.com>
wrote:

> In app/pages/patientdata/patientdata.less
> <https://github.com/tidepool-org/blip/pull/182#discussion-diff-22882329>:
>
> > @@ -33,16 +33,21 @@
> >
> >  .patient-data-subnav-text {
> >    display: inline-block;
> > -  padding: @patient-data-subnav-vertical-padding @patient-data-subnav-horizontal-padding;
> > +  padding: @patient-data-subnav-vertical-padding 0 @patient-data-subnav-horizontal-padding 0;
>
> There's a funny thing going one when the app is in the loading data state
> (title bar gets really vertically skinny), and I think this change might be
> the root?
> [image: screenshot 2015-01-13 10 19 44]
> <https://cloud.githubusercontent.com/assets/1588547/5726139/f5f1965c-9b0d-11e4-859f-daa750ee84a0.png>
>
> To get a good long look at it you can use "mock" mode and and the URL
> http://localhost:3000/?api.patientdata.get.delay=5000#/patients/11/data
> (auth.skip in mock mode seems to be broken though...)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tidepool-org/blip/pull/182/files#r22882329>.
>